src/main/java/org/rcsb/cif/CifOptions.java

```diff
@@ -283,8 +283,17 @@ public CifOptionsBuilder columnBlacklist(List<String> columnBlacklist) {
             return this;
         }
 
-        private static final Gson GSON = new Gson();
-        private static final Type LIST_TYPE = new TypeToken<ArrayList<EncodingStrategyHint>>(){}.getType();
+        // Lazy initialization if no JSON is desired
+        
+        private static Gson GSON;
+        private static Gson getGson() {
+        	return (GSON == null ? GSON = new Gson() : GSON);
+        }
+        
+        private static Type LIST_TYPE;        
```

Comment:
Package-private static inner class holding static reference.

Reply:
Sure - whatever works to not have it part of the static initializer of the class. You can close the pull request any time.

Reply:
`static class SingletonHolder {
  static Singleton instance = new Singleton();    
}
public static Singleton getInstance() {
  return SingletonHolder.instance;
}`

Yes, that's quite nice. I had not thought of that, but sure. The class will only be statically initialized upon the first attempt to read the static field. Very slick, very conscise! Also, I would recommend just making that a static _inner_ class of CifOptions.